src/hooks/useTransactionFlow.ts

```diff
@@ -71,6 +73,7 @@ export const useTransactionFlow = () => {
         quantity: 1,
         amount: quote?.digital_asset_amount,
         wallet: account?.address,
+        integrator,
```

Comment:
```suggestion
        integrator: some_value,
```
Shouldn't it be something like `integrator: integratorValueSlug`? Maybe I'm missing something

Reply:
here the value is taken from the config object (required field when you integrate the widget), and in js if the value and key is the same, you can write it like that, so:
 
 `integrator: integrator`= `integrator`

Reply:
oh got it! thanks